src/server/models/Meter.js

```diff
@@ -177,15 +179,15 @@ class Meter {
 	 * Updates the meter with values passed if not undefined.
 	 * For parameter info see {@link Meter#constructor}.
 	 */
-	merge(name = this.name, ipAddress = this.ipAddress, enabled = this.enabled, displayable = this.displayable, type = this.type,
+	merge(name = this.name, url = this.url, enabled = this.enabled, displayable = this.displayable, type = this.type,
 		meterTimezone = this.meterTimezone, gps = this.gps, identifier = this.identifier, note = this.note, area = this.area,
 		cumulative = this.cumulative, cumulativeReset = this.cumulativeReset, cumulativeResetStart = this.cumulativeResetStart,
 		cumulativeResetEnd = this.cumulativeResetEnd, readingGap = this.readingGap, readingVariation = this.readingVariation,
 		readingDuplication = this.readingDuplication, timeSort = this.timeSort, endOnlyTime = this.endOnlyTime,
 		reading = this.reading, startTimestamp = this.startTimestamp, endTimestamp = this.endTimestamp) {
 
 		this.name = name;
-		this.ipAddress = ipAddress;
+		this.url = rul;
```

Comment:
Is "rul" a typo?

Reply:
Fixed. I note the same fix is in PR #781 where it was also found.